ivy/functional/ivy/linear_algebra.py

```diff
@@ -1894,10 +1894,97 @@ def solve(
     extensions/generated/array_api.linalg.solve.html>`_
     in the standard.
 
-    Both the description and the type hints above assumes an array input for simplicity,
+    Both the description and the type hints above assume an array input for simplicity,
     but this function is *nestable*, and therefore also accepts :class:`ivy.Container`
     instances in place of any of the arguments.
 
+  ------
+  
+  Function Examples:
+
+  With class:`ivy.Array` input:
+
+  - For shape(A) = (3,3) and shape(B) = (3,1):
+    
+    >>> A = ivy.array([[1.1, 1.2, 1.3],
+                       [2.1, 2.2, 2.3],
+                       [3.1, 3.2, 3.3]]),
+    >>> B = ivy.array([[1.1],
+                       [2.1],
+                       [3.1]]),
+    >>> x = solve(A,B);
+    >>> print(x)
+    >>> ivy.array([[1],
+                   [0],
+                   [0]])
+    >>> print(x.shape)
+    (1,3)
+    
+    (The simplest case in which Ax=B)
+
+  - For shape(A) = (2,3,3) and shape(B) = (2,3,1):
+    
+    >>> A = ivy.array([[[11.1, 11.2, 11.3],
+                        [12.1, 12.2, 12.3],
+                        [13.1, 13.2, 13.3]],
+                       [[21.1, 21.2, 21.3],
+                        [22.1, 22.2, 22.3],
+                        [23.1, 23.2, 23.3]]
+                        ]),
+    >>> B = ivy.array([[[11.1],
+                        [12.1],
+                        [13.1]],
+                       [[21.1],
+                        [22.1],
+                        [23.1]]]),
+    >>> x = solve(A,B);
+    >>> print(x)
+    ivy.array([[[1],
+                [0],
+                [0]],
+               [[1],
+                [0],
+                [0]]])
+    >>> print(x.shape)
+    (2,1,3)
+
+    (A and B are both multi-dimensional matrices. A(1,:,:)x(1,:,:) = B(1,:,:), and A(2,:,:)x(1,:,:) = B(2,:,:).)
```

Comment:
these shouldn't be added in between example as it causes difficulties with docstring tests. Instead could you either add this above in description or add it as a python comment along with code.

Reply:
Do you mean remove the explanations in brackets in between examples? If so I've deleted them. Please have a another review!